docs/source/release_roadmap.org

```diff
@@ -0,0 +1,363 @@
+certain effect. As such, smartcontracts which link against the same
+_resident_ library dynamically (or statically linked at compile time)
+become portable across blockchains, provided they can be executed with
+the provided virtual machine. Further, the data model becomes an
+implementation detail for both the clients which were used at the
+submission time of the instructions and the Executor. The parts of the
+data model which affect the communication of the Executor to the host
+node remain frozen and thus don't need to be updated. Because
+instructions no longer enforce any of the blockchain invariants, this
+is not restrictive and a natural method of extension is provided by
+extending the Executor with custom expressions.
+
+No further benefits.
+
+* Scenario 3: ISA refactor and change of smartcontract language
+
+Depending on which smartcontract language is realised, the technical
+limitations incurred by usage of WASM can either be eliminated or
+substantially reduced. The precise character of the reductions shall
+be outlined in the RFC. But the short version follows: standards with
+guaranteed backwards compatibility, such as Java, native C-ABI
+libraries, and an embedded scripting language all eliminate the need
+for hosting multiple versions of multiple machines for validation.
+This greatly simplifies deployment of new nodes, attracting more users
+in the public blockchain space, as well as reducing the operational
+costs to private blockchains.
+
+Removal of WASM, will definitely eliminate the need to implement
+shared-memory dynamic linkage, and instead use the standard-native
+protocol. C-ABI dynamic linkage is implemented by the operating
+system, resulting in the best outcome. For Java some bridging code may
+be needed. For =lua= and Python, the dynamic linkage is an
+implementation-specific detail of the interpreter, and only minor
+adjustments may be needed to use the exported functions via a
+Foreign-function interface. For the record the same work must be done
+for WASM to reduce the size of smartcontracts, which is not a
+consideration on a private blockchain, but might become a problem in a
+public use-case.
+
+Depending on the specific choice, the popularity might either be
+improved substantially or be at least as good as with WASM. C-ABI
+dynamic linkage offers the widest compatibility, as it is a ubiquitous
+operating-system-guaranteed interface. One could, in principle write
+smartcontracts in regular =x86= assembly to offer the best size
+optimisations. This can be combined with a scripting language also
+being native supported on-chain, because languages like =lua= and
+Python offer support for native linkage as part of the standard
+interpreter. The JVM would offer a wider range of compatible programs,
+and though the full runtime library might be in excess of a few
+hundred megabytes, this cost must be paid once up-front and provide
+compatibility with a wide range of languages
```

Comment:
```suggestion
compatibility with a [wide range of languages](https://en.wikipedia.org/wiki/List_of_JVM_languages).
```

Reply:
@WRRicht3r This is an org document, the link syntax is slightly different: 
```org
[[link text][link destination]]
```